src/api/resources/sample_units/benthicphotoquadrattransectmethod.py

```diff
@@ -154,7 +153,7 @@ class Meta(BaseSUViewAPISerializer.Meta):
         )
 
 
-class BenthicPQTMethodGeoSerializer(BaseViewAPIGeoSerializer):
+class BenthicPQTObsMethodGeoSerializer(BaseViewAPIGeoSerializer):
```

Comment:
I know it doesn't matter, but better to be consistent from the start... To be consistent this would be `BenthicPQTMethodObsGeoSerializer` and not `BenthicPQTObsMethodGeoSerializer`

Reply:
sorry :(